src/cephadm/cephadm

```diff
@@ -3071,6 +3071,47 @@ def install_sysctl(ctx: CephadmContext, fsid: str, daemon_type: str) -> None:
         call_throws(ctx, ['sysctl', '--system'])
 
 
+def migrate_sysctl_dir(ctx: CephadmContext, fsid: str) -> None:
+    """
+    Cephadm once used '/usr/lib/sysctl.d' for storing sysctl configuration.
+    This moves it to '/etc/sysctl.d'.
+    """
+    deprecated_location: str = '/usr/lib/sysctl.d'
+    deprecated_confs: List[str] = glob(f'{deprecated_location}/90-ceph-{fsid}-*.conf')
+    if not deprecated_confs:
+        return
+
+    file_count: int = len(deprecated_confs)
+    logger.info(f'Found sysctl {file_count} files in deprecated location {deprecated_location}. Starting Migration.')
+    for conf in deprecated_confs:
+        try:
+            shutil.move(conf, ctx.sysctl_dir)
+            file_count -= 1
+        except shutil.Error as err:
+            if str(err).endswith('already exists'):
+                logger.warning(f'Destination file already exists. Deleting {conf}.')
+                try:
+                    os.unlink(conf)
+                    file_count -= 1
+                except OSError as del_err:
+                        logger.warning(f'Could not remove {conf}: {del_err}.')
```

Comment:
```
cephadm:3405:25: E117 over-indented
1     E117 over-indented
```
Line number looks different because it is post merging these changes with master branch, but I'm pretty sure it's referring to this line being over indented. After that's fixed, we should just need to put this through testing one last time and then we can merge @windgmbh

Reply:
@adk3798 You are absolutely correct. Not sure how this has happened, because I must have used `mypy` to test the code. However, it is fixed now. Sorry for this rookie mistake.